### ISSUE TITLE
Make mobile UI elements smaller

### ISSUE THREAD

**Issue by SievaFliesAlone** [KenEucker/biketag-vue#123]
In the old app, UI elements on mobile were smaller and fit the screen better. Now, in my opinion, everything is too big, which results in less useful info on the screen.

![Screenshot_20220315-071035_Fennec](https://user-images.githubusercontent.com/790717/158397687-4697f4b6-b95e-4fa1-b436-65725313f8c6.png)
![Screenshot_20220315-071110_Fennec](https://user-images.githubusercontent.com/790717/158397707-e6045b39-aede-4b03-9e2f-46ed06c289dc.png)
![Screenshot_20220315-071809_Fennec](https://user-images.githubusercontent.com/790717/158398199-4baf9e5d-bf2e-411b-ac97-8cedf0fb45c8.png)
![Screenshot_20220315-071842_Fennec](https://user-images.githubusercontent.com/790717/158398221-13a93f41-366a-4cc7-babd-83c595b13fca.png)

**Comment by KenEucker** [comment]
@anibyl I have removed the bottom links as you suggested. That change will go out sometime in the next day or two.

What this project really needs is someone a bit stronger in the ways of CSS to make everything look really good. I must admit that frontend development has not been my strongest domain but I always do my best to incorporate the suggestions of others and I try to foster a development environment that others can easily get started with to make contributions themselves. 

I don't personally have the time or money to pay for more development resources or to continue doing project management for these bodies of code that comprise BikeTag 2.0, at least not until next winter. I am hoping that this project will see more investment in the time between now and then so we can look forward to having an opportunity to build the yet-to-be-started features that this new platform was intended to introduce as opposed to supporting two separate versions of the game on two different platforms.

**Comment by ykyu** [comment]
Un-sticking the header would help a lot too. There's not a ton of benefit to sticking it since all of the menu items other than "About" are already on the home page, and the pages are small enough that scrolling up to go "Home" isn't a huge deal. Even adding the footer links (Top 10, Players) to the menu doesn't make it important enough to stick, imo.

I'm not familiar enough with Vue or TS to contribute directly, but here are messy adjustments that can help improve UI a little? idk.

**Un-stick header**
In index.css, remove from "header[data-v-ac0989c4]" `position:fixed;`
In the HTML for the header, remove `<div class="spacer-top"></div>`

**Reduce margin between home page in-line menu (Biketags, PLAY!, How-to) and current bike tag**
I couldn't figure out why .play-screen__mystery-player has such a big top margin, so this is probably an inelegant fix, but:

Home.css, add:
`.play-screen__mystery-player, .play-screen { top: -50px; }`

**Comment by KenEucker** [comment]
@anibyl I'll let you close your own issue here.

This has been resolved.

**Comment by SievaFliesAlone** [comment]
It's much better now, thanks! Closing the issue.

_However, I had to clear cache in both of my browsers in order for the app to work._

**Comment by KenEucker** [comment]
@anibyl yes, that is a known issue. I am hoping, however, that this last update you've done comes with an easier process going forward. Please do create a new issue, next week, if this issue persists for you.
